### ISSUE TITLE
Find should be able to whitelist translation files like extract does

### ISSUE THREAD

**Issue by MikeDabrowski** [ngneat/transloco-keys-manager#52]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Performance issue
[x] Feature request
[ ] Documentation issue or request
[ ] Support request
[ ] Other... Please describe:
</code></pre>

## Current behavior

`Find` check all json files in translations directory. Our client wanted to be able to see translation keys to help him do the translations so we used empty json file as one of the steps to enable such feature. Later we realized that using find now produces huge output detailing that all keys are missing from that empty json.  I'm writing this as a feature request to just add similar config option as extract has, where we whitelist desired languages to check. However if there is some better way to toggle translation to translation keys (probably directly in transloco) please let me know.

## Expected behavior

`Find` should have an option to whitelist languages that will be checked.

## Minimal reproduction of the problem with instructions

Just add empty.json file next to files with translations and run 'find'

## What is the motivation / use case for changing the behavior?

See at the beginning 

## Environment

<pre><code>
Angular version: ~9.1.0
transloco: ^2.14.2,

For Tooling issues:
- Node version: 12.16.3
- Platform:  Windows 

</code></pre>

**Comment by shaharkazaz** [comment]
@MikeDabrowski Currently I don't see how many users will need such a feature. as we discussed I recommend you create a fork at the time being and implement the logic you need there.
If I see that there is a growing demand we can always reconsider adding this into the project.

**Comment by MikeDabrowski** [comment]
A bit disappointing, esp. given that this would be basically a filter on the files, but ok - I will manage this locally then.

**Comment by shaharkazaz** [comment]
@MikeDabrowski If you are already implementing this you can create a PR for it and we can see how we push this forward.
What's the property you had in mind adding? what's the behavior? passing the langs you want to skip when using find?

**Comment by MikeDabrowski** [comment]
Similar to extract: I think it was `-l en fr de` to whitelist. Then in the code I tried to add filter that checks if the file found in the directory is in the whitelist and filter out rest. I have not yet succeeded in testing it out on any of my projects.

**Comment by shaharkazaz** [comment]
@MikeDabrowski I see, but it's not exactly the same, this filter in the extract is meant to instruct the manager which languages files to generate.
I think that in the `find` this might be a bit odd, we need to think about how we want to expose such functionality.
but first, make it work on your projects and we can continue from there 🙂 
Assigning this issue to you.

**Comment by MikeDabrowski** [comment]
I agree it will be different. Of course that was only my initial idea, I thought it would be the quickest and least code. 
On the other hand, why would `find` read every json in lang directory?

**Comment by shaharkazaz** [comment]
@MikeDabrowski Since they are different files, each one can have a different set of keys.
The `find` command job is to show you inconsistency.

**Comment by MikeDabrowski** [comment]
I managed to do the changes described above with just a slight modification of compare-keys-to-files. The change is subtle, and since there is no demand yet, I didn't bother with the PR for now (but I'm happy to make one)

https://github.com/ngneat/transloco-keys-manager/blob/9affd8edebdf3520c9f499b73de88325d8ca7506/src/keys-detective/compare-keys-to-files.ts#L137-L140

```typescript
  const langsFromConfig = getConfig().langs;
  const langs = Object.keys(diffsPerLang).filter((lang) => {
    if(!langsFromConfig.includes(lang)) {
       return false;
    }
    const { missing, extra } = diffsPerLang[lang];
    return missing.length || extra.length;
  });

```

The usage is `transloco-keys-manager find -l en-GB nl-NL de-DE de-CH ch-CH fr-FR` - so it reuses the existing code for CLI options and `-l` option to specify which langs to check. 

NOTE:
Key detective will search all files detected under specified path and produce diffs for each. The change here is just to prevent unwanted langs from being printed to the console.

@shaharkazaz What do you think?
